### ISSUE TITLE
Public Key Appears to Only Support OpenSSH

### ISSUE THREAD

**Issue by alph486** [sequenceiq/cloudbreak#310]
TL;DR:
Two potential issues:
1. The way I read the docs it mentions the public keys for AWS are not required, however I cannot submit a new credential without specifying them.
2. If they are required, I cannot use my AWS generated public key or a local RSA key because only OpenSSH is supported. 

More:
The library used in CloudBreak to validate public SSH keys used in new AWSCredential objects (https://github.com/fommil/openssh-java) appears to only support OpenSSH type keys. This may be by design as it is described for the Google and Azure accounts here: http://sequenceiq.com/cloudbreak/#quickstart-and-installation.  

Even if this is by design may be good to support SSH2 public keys though. AFAIK, the keys generated by AWS are SSH2 and most local client generated keys will be generated by ssh-keygen which supports SSH type 2. In this case this is more of a feature request.

**Comment by doktoric** [comment]
fixed for amazon credentials
